com.unity.render-pipelines.high-definition/Documentation~/IES-Importer.md

```diff
@@ -0,0 +1,25 @@
+# IES Importer
+
+![](Images/IES-Importer1.png)
```

Comment:
Documentation for this one should be in core, not in HDRP

Reply:
(and we should have a link to core somewhere (maybe on IES property of light)

Reply:
Currently this one only exist on HDRP. The code on Core is kind of "partial class".

Reply:
ok